backend/corpora/lambdas/api/v1/curation/collections/collection_uuid/actions.py

```diff
@@ -16,3 +19,16 @@ def delete(collection_uuid: str, token_info: dict):
     else:
         collection.delete()
     return "", 204
+
+
+@dbconnect
+def get_collection_uuid(collection_uuid: str, token_info: dict):
```

Comment:
nit: this could just be `get`, the function is clear from the filepath. Would conform to collection uuid `delete` action's naming convention, too.